src/site/content/en/blog/carousel-best-practices/index.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title: Best practices for carousels
+subhead: Optimize carousels for performance and usability.
+authors:
+  - katiehempenius
+description:  Learn how to optimize carousels for performance and usability.
+date: 2021-01-26
+hero: hero.jpeg
+tags:
+  - blog
+  - fast
+  - performance
+  - web-vitals
+  - internationalization
+---
+
+A carousel is a UX component that displays content in slideshow-like manner. Carousels can "autoplay" or be navigated manually by users. Although carousels can be used elsewhere, they are most frequently used to display images, products, and promotions on homepages.
+
+## Performance
+
+This section discusses the performance impact of carousels and best practices for building performant carousels.
+
+
+### Impact
+
+A well-implemented carousel, in and of itself, should have very minimal or no impact on performance. That being said, carousels often contain large media assets. Large assets can impact performance regardless of whether they are displayed in a carousel or elsewhere.
+
+
+*   **LCP**
+
+    Large, above-the-fold carousels often contain the page's LCP element, and therefore can have a significant impact on LCP. In these scenarios, optimizing the carousel may significantly improve LCP.
+
+    For an in-depth explanation of how LCP measurement works on pages containing carousels, refer to the Appendix.
+
+*   **FID**
+
+    Carousels have minimal JavaScript requirements and therefore should not impact page interactivity. If you discover that your site's carousel has long-running scripts, you should consider replacing your carousel tooling.
+
+*   **CLS**
+
+    A surprising number of carousels use janky, non-composited animations that can contribute to CLS. On pages with autoplaying carousels, this has the potential to cause infinite CLS. This  type of CLS typically isn't apparent to the human eye, which makes this issue easy to overlook. To avoid this issue, [avoid using non-composited animations](https://web.dev/non-composited-animations/) in your carousel (for example, during slide transitions).
```

Comment:
```suggestion
    A surprising number of carousels use janky, non-composited animations that can contribute to CLS. On pages with autoplaying carousels, this has the potential to cause infinite CLS. This type of CLS typically isn't apparent to the human eye, which makes the issue easy to overlook. To avoid this issue, [avoid using non-composited animations](/non-composited-animations/) in your carousel (for example, during slide transitions).
```